### ISSUE TITLE
If / then / endif conversion failure

### ISSUE THREAD

**Issue by dickvand** [kobbled/tp_plus#14]
The conversion of
```
if foo == 1 then
 # do something
end
```
gives an ENDIF with two semicolons:
```
 : IF (R[1:foo]=1) THEN ;
 : ! do something ;
 : ENDIF ; ;
```

That second semicolon gives an error when importing the file to Fanuc.

**Comment by kobbled** [comment]
Man, so sorry about that one. I didn't have good test coverage for that. Its fixed now, dfb1a4a. Thank you for your patience and posting the issues. I've been side tracked by other things recently so I'm not coding enough in tp+ to discover my own bugs apparently.

**Comment by dickvand** [comment]
That is totally no problem, I'm happy to help.
Thank you for fixing the issues so quickly!
________________________________
Van: Matt Dewar ***@***.***>
Verzonden: donderdag 22 juli 2021 21:20
Aan: kobbled/tp_plus ***@***.***>
CC: dickvand ***@***.***>; Author ***@***.***>
Onderwerp: Re: [kobbled/tp_plus] If / then / endif conversion failure (#14)


Man, so sorry about that one. I didn't have good test coverage for that. Its fixed now, dfb1a4a<https://github.com/kobbled/tp_plus/commit/dfb1a4a625607360bc8854a9405fc345e89a9416>. Thank you for your patience and posting the issues. I've been side tracked by other things recently so I'm not coding enough in tp+ to discover my own bugs apparently.
